plugin-gradle/src/main/java/com/diffplug/gradle/spotless/FormatExtension.java

```diff
@@ -462,6 +462,12 @@ public LicenseHeaderConfig yearSeparator(String yearSeparator) {
 			return this;
 		}
 
+		public LicenseHeaderConfig skipLinesPattern(String skipLinesPattern) {
```

Comment:
Maybe `skipLinesMatching`? I'm okay with the current name too, author's choice :)

Reply:
That's sounds better. :)
Changed.